File: FishBunDemo/src/main/java/com/sangcomz/fishbundemo/WithFragmentActivity.java
package com.sangcomz.fishbundemo;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.widget.RelativeLayout;

public class WithFragmentActivity extends AppCompatActivity {

    RelativeLayout areaContainer;
    SubFragment subFragment;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_withfragment);

        areaContainer = (RelativeLayout)findViewById(R.id.area_container);
        subFragment = new SubFragment();

        getSupportFragmentManager().beginTransaction().add(areaContainer.getId(), subFragment).commit();

    }

    /**
     * Send onActivityResult method to SubFragment
     */
    @Override
    protected void onActivityResult(int requestCode, int resultCode, @Nullable Intent data) {
        subFragment.onActivityResult(requestCode, resultCode, data);
    }
}

File: FishBunDemo/src/main/java/com/sangcomz/fishbundemo/WithFragmentActivity.kt
package com.sangcomz.fishbundemo

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.sangcomz.fishbundemo.databinding.ActivityWithfragmentBinding

class WithFragmentActivity : AppCompatActivity() {

    private lateinit var subFragment: SubFragment
    private lateinit var binding: ActivityWithfragmentBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityWithfragmentBinding.inflate(layoutInflater)
        setContentView(binding.root)

        subFragment = SubFragment.newInstance()
        supportFragmentManager.beginTransaction().add(binding.areaContainer.id, subFragment)
            .commit()
    }
}
